proxy/src/bin/proxy.rs

```diff
@@ -357,7 +373,7 @@ async fn main() -> anyhow::Result<()> {
 
     let cancel_map = CancelMap::default();
 
-    let redis_publisher = match &regional_redis_client {
+    let redis_publisher = match &redis_notifications_client {
```

Comment:
I think I am missing context around this specific change

Reply:
Thank you!